Package.swift

```diff
@@ -0,0 +1,35 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "CocoaLumberjack",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "CocoaLumberjack",
+            type: .dynamic,
```

Comment:
CocoaLumberjack also supports static linking I think. Could remove this and let clients decide?

Reply:
If `Swift Package manager` supports static linking, then, we should add static library also.

Reply:
> /// A library's product can either be statically or dynamically linked. It
> /// is recommended to not declare the type of library explicitly to let the
> /// Swift Package Manager choose between static or dynamic linking depending
> /// on the consumer of the package.

https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescription.md
Removing it should be fine?